src/components/v2/V2Project/NewDeployModal.tsx

```diff
@@ -87,7 +80,7 @@ export default function NewDeployModal({
         />
         <RichButton
           prefix="2"
-          heading={<Trans>Create a payable address</Trans>}
+          heading={<Trans>Create a payable address (optional)</Trans>}
           description={
             <Trans>
               Create an Ethereum address that can be used to pay your project
```

Comment:
'pay your project directly' is kind of unclear to me, is paying through the UI not also 'paying the project directly'? WDYT about

'Create an Ethereum address to receive payments to your project. Enables contributors to pay without going through your project's Juicebox page.' 

or something similar?

Reply:
mmm, i think its pretty clear without being too long. lets handle in follow-up if you feel strongly

Reply:
Aactually i like yours. sgtm